app/models/concerns/conclusion_reviews/upl_pdf.rb

```diff
@@ -43,6 +44,14 @@ def put_upl_cover_on pdf, organization
       put_upl_review_owners_on pdf
     end
 
+    def upl_business_unit_tags
+       review.plan_item.business_unit&.tags.join
```

Comment:
Esto siempre es una de momento, pero un join así no agrega un espacio, pondría algo así:
```suggestion
       review.plan_item.business_unit&.tags.to_sentence
```